### ISSUE TITLE
What does "an interoperable way to request media and encodings" actually mean?

### ISSUE THREAD

**Issue by vasilvv** [moq-wg/moq-charter#43]
I am not entirely certain by what's implied by that.  "requesting media" feels like something that would belong in the name/receive protocol, since requesting something requires a name?

**Comment by kixelated** [comment]
I agree; it's too vague. I would even think establishing a connection is enough to "request media".

I'm not sure if the draft is trying to imply pull versus push and/or if there's some capabilities exchange. But I would argue that neither are required, as they can be handled by a higher layer (similar to WebRTC).
